xgcm/test/test_metrics.py

```diff
@@ -311,28 +307,94 @@ def test_set_metric():
         assert k in grid_manual._metrics.keys()
 
         for metric_expected, metric in zip(v, grid_manual._metrics[k]):
-            xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+            xr.testing.assert_allclose(metric_expected.reset_coords(drop=True), metric)
 
         for metric_expected, metric in zip(v, grid._metrics[k]):
-            xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+            xr.testing.assert_allclose(metric_expected.reset_coords(drop=True), metric)
 
 
-@pytest.mark.skip(reason="For the next PR")
 @pytest.mark.parametrize(
-    "metric_axes,metric_name",
+    "metric_axes, metric_varname, overwrite_",
     [
-        ("X", "dx_t"),
-        ("Y", "dy_ne"),
+        ("X", ["dx_n", "dx_t", "dx_ne", "dx_t"], True),
```

Comment:
I might split this into two tests: One for overwrite=True and another for overwrite=False. Each of these could test that the appropriate warning/error is raised each time (see my comment above). We can discuss this later too.